sdk/storage/test-resources.json

```diff
@@ -420,6 +420,18 @@
             "type": "string",
             "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('primaryAccountName')), variables('storageApiVersion')).keys[0].value]"
         },
+        "PRIMARY_STORAGE_BLOB_ENDPOINT": {
+          "type": "string",
+          "value": "[reference(resourceId('Microsoft.Storage/storageAccounts', variables('primaryAccountName')), variables('storageApiVersion')).primaryEndpoints.blob]"
+        },
+        "PRIMARY_STORAGE_FILE_ENDPOINT": {
+          "type": "string",
+          "value": "[reference(resourceId('Microsoft.Storage/storageAccounts', variables('primaryAccountName')), variables('storageApiVersion')).primaryEndpoints.file]"
+        },
```

Comment:
these are for batch and share samples. we won't read them in TestEnvironment until necessary (but we'll know how to do it). This will become important when testing in other clouds will be a thing.